frontend/lib/ui/Button/Button.module.css

```diff
@@ -73,7 +73,8 @@
 
   &.sm {
     font-size: 14px;
```

Comment:
why are sizes defined in `px` and not in `rem`?
16px = 1rem so 14px = `0,875rem`